.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "packages/uikit"]
+	path = packages/uikit
+	url = git@github.com:sendbird/sendbird-uikit-react.git
+	branch = ai-widget/experimental
```

Comment:
what is this file for?

Reply:
Related to the things I mentioned in the PR description. 
<img width="822" alt="Screenshot 2024-04-30 at 3 39 04 PM" src="https://github.com/sendbird/chat-ai-widget/assets/10060731/978a8770-8060-406a-ab6d-9cbfd33a65d0">

Let me know if you need more detailed explanation :)